hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/RecordAssertions.java

```diff
@@ -77,7 +77,6 @@ public static void triggerAndCloseAtLeastOneFile(final HapiSpec spec) throws Int
                 .hasAnyStatusAtAll()
                 .noLogging();
         allRunFor(spec, triggerOp);
-        Thread.sleep(DEFAULT_RECORD_CLOSE_DELAY.toMillis());
```

Comment:
No longer needed because?

Reply:
I'm not sure why we had two sleeps here. 🤔